### ISSUE TITLE
[Adaptive Cards iOS - Container.SelectAction.json]: Submit action is look like normal text instead of button.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8412]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Container.SelectAction.json' under 'Elements' list item and activate it.
7. Card will appear.
8. Navigate to 'Submit action' and double tap on it.
9. Observe the issue.

**Actual Result:**
Submit action looks like normal text instead of button and **'Cool Link and Emaphasis link'** are looking like a normal text instead of 'link'.

**Expected Result:**
Submit action should be visible as button and 'Emaphasis link' and 'Cool link' should be visible as 'link' with underline.

**Notes:**
v1.0-Elements-ColumnSet.SelectAction.json: 'Map', 'With Emphasis' and 'Netflix' is look like normal text instead of link.
Please refer attachment: "Note-'Map', 'With Emphasis' and 'Netflix' is look like normal text instead of link.png"

**User Experience:**
Low vision user will be impacted if Submit action is look like normal text instead of button.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/227934304-b6c070d6-f59b-4838-8311-fe95af4dbd33.MP4

<img width="960" alt="Note-'Map', 'With Emphasis' and 'Netflix' is look like normal text instead of link" src="https://user-images.githubusercontent.com/95913748/227934329-ed72aab8-361b-45e7-8dc2-844fd6f85f77.png">

### Card JSON

```JSON
NA
```
